api/database/migrations/2022_06_30_182650_update_pool_with_advertisement.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class UpdatePoolWithAdvertisement extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('pools', function (Blueprint $table) {
+            $table->timestamptz('expiry_date')->nullable();
+        });
+
+        Schema::table('pools', function (Blueprint $table) {
+            $table->boolean('is_published')->nullable();
```

Comment:
Is there a case where `is_published` will be `null`?
```suggestion
            $table->boolean('is_published')->default(false);
```